SPLASH/src/game/states/LobbyState.cpp

```diff
@@ -210,7 +209,7 @@ bool LobbyState::renderImgui(float dt) {
 	renderPlayerList();
 
 	// ------- CHAT LOG ------- 
-	renderChat();
+	//renderChat();
```

Comment:
remove comment.
Also remove the whole renderChat function from Lobbystate so we dont have unused code.